bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/XmlNonElementItem.java

```diff
@@ -141,8 +146,11 @@ public BXml descendants() {
     public XmlValue getItem(int index) {
         if (index == 0) {
             return this;
+        } else if (index > 0) {
```

Comment:
Can get rid off the else.
```suggestion
        } 
        
        if (index > 0) {
```

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/40114/files#r1191913089

Reply:
Updated. 5ff4ac52aa7ec017f6582c9daaf91208f3566c3c